app/models/task.py

```diff
@@ -3,3 +3,23 @@
 
 class Task(db.Model):
     task_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, default=None)
+    goal_id = db.Column(db.Integer, db.ForeignKey("goal.goal_id"), nullable=True)
+    goal = db.relationship("Goal", back_populates="tasks")
+    
+    def to_dict(self): 
+        task_dict={
+        "id": self.task_id,
+        "title": self.title,
+        "description": self.description,
+        "is_complete": False,
```

Comment:
Instead of hardcoding `is_complete` to `False`, how could you use the value of `self.completed_at` to derive the boolean value?

We could use a ternary operator like: 
```python
task_dict["is_complete"] = True if self.completed_at else False